bin/lib/couchdb.js

```diff
@@ -0,0 +1,151 @@
+
+/*
+Copyright (c) 2012 Adobe Systems Incorporated. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
Nope, it's just the license this file had. It's this file: https://github.com/apache/cordova-medic/blob/master/src/couchdb/interface.js. What should the license be?